Dockerfile

```diff
@@ -1,10 +1,28 @@
 FROM alpine:3.4
 
-MAINTAINER Kevin Smith <kevin@operable.io>
+MAINTAINER Christopher Maier <christopher.maier@gmail.com>
 
-# Bake in a directory that we can use for logging, config, etc.
-RUN mkdir -p /var/operable/relay
+ENV GO_PACKAGE_VERSION 1.6.3-r0
+ENV GOPATH /gopath
+ENV PATH=${GOPATH}/bin:${PATH}
 
-# Relies on the binary having already been built
-COPY _build/relay /usr/local/bin
-COPY docker/relay.conf /usr/local/etc/relay.conf
+WORKDIR /gopath/src/github.com/operable/go-relay
+COPY . /gopath/src/github.com/operable/go-relay
+
+RUN apk -U add \
```

Comment:
@jesselang I'll go ahead and add the virtual package if for no other reason than to promote good patterns via copy and paste 😄 

I'd forgotten about removing the APK cache; I'll give that a try this evening and see if it helps. Either way, I'll merge it afterward.

Thanks for the review!